src/App.tsx

```diff
@@ -34,6 +36,26 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort((a, b) => a.length - b.length);
```

Comment:
1. Follow DRY, sort just once & use switch in callback
2. You can return imideately from case